.github/workflows/build_cmake.yml

```diff
@@ -153,18 +155,15 @@ jobs:
     - name: CMake environment
       shell: bash --noprofile --norc -eo pipefail -x {0}
       run: |
-        echo "::add-path::$GITHUB_WORKSPACE"
-        echo "::set-env name=CCACHE_BASEDIR::$GITHUB_WORKSPACE"
-        echo "::set-env name=CCACHE_DIR::$GITHUB_WORKSPACE/.ccache"
-        echo "::set-env name=cmake_args::\
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DENABLE_PCH=OFF"
+        echo "$GITHUB_WORKSPACE" >> $GITHUB_PATH
+        echo "CCACHE_BASEDIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+        echo "CCACHE_DIR=$GITHUB_WORKSPACE/.ccache" >> $GITHUB_ENV
+        echo "cmake_args='-DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DENABLE_PCH=OFF'" >> $GITHUB_ENV
 
     - name: Configure (Linux)
       if: runner.os == 'Linux'
       shell: bash --noprofile --norc -eo pipefail -x {0}
-      run: cmake $cmake_args -H. -Bbuild -GNinja -DCMAKE_BUILD_TYPE=${{ matrix.config.build_type }}
+      run: cmake $cmake_args -H. -Bbuild -GNinja -DPORT=Qt -DCMAKE_BUILD_TYPE=${{ matrix.config.build_type }}
```

Comment:
It would be better to bring back 4ba116a15beca3157a9b2b5854a0b54ab1bf8575 and avoid using `-PORT=Qt` everywhere. Also, users would be less eager to trying building with different port value (which is likely going to fail)